src/vcSettings.cpp

```diff
@@ -822,3 +827,24 @@ udResult vcSettings_UpdateLanguageOptions(vcSettings *pSetting)
 epilogue:
   return result;
 }
+
+void vcSettings_UpdateServerFlag(vcSettings *pSettings)
+{
+    UDCOMPILEASSERT(udLengthOf(g_vcMapTileServerName) == vcMTSF_Count, "Update array");
```

Comment:
This seems an unusual place for the compile assert, since it runs at compilation time you can place it anywhere and it shouldn't make a difference, but generally elsewhere in the code we have them immediately after the definition of the array or enum

Reply:
Ok, thanks for your comments.

Reply:
All right,Assertions have been placed where variables are declared.